packages/office-ui-fabric-react/src/common/_semanticSlots.scss

```diff
@@ -30,6 +30,7 @@ $inputFocusBorderAltColor: "[theme:inputFocusBorderAlt, default: #0078d7]";
 $menuItemBackgroundHoveredColor: "[theme:menuItemBackgroundHovered, default: #f8f8f8]";
 $menuItemBackgroundCheckedColor: "[theme:menuItemBackgroundChecked, default: #d0d0d0]";
 $menuIconColor: "[theme:menuIcon, default: #0078d7]";
+$menuCheckmarkIconColor: "[theme:menuCheckmarkIcon, default: #666666]";
```

Comment:
instead of adding a new slot, reuse bodySubtext
(but thank you for being so thorough with adding the new slot!)